microservices/realtime/pkg/services/http.go

```diff
@@ -0,0 +1,86 @@
+package services
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"irontec.com/realtime/pkg/config"
+	"irontec.com/realtime/pkg/utils"
+)
+
+var transport = &http.Transport{
+	TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+}
+
+// Define a global HTTP client
+var client = &http.Client{Transport: transport}
+
+type CriteriaResponse struct {
+	Criteria string `json:"criteria"`
+}
+
+func GetActiveCallsFilter(token string, role string) (*string, error) {
+	var api string
+
+	switch role {
+	case utils.ROLE_SUPER_ADMIN:
+		api = config.GetHttpApiPlatform()
+	case utils.ROLE_BRAND_ADMIN:
+		api = config.GetHttpApiBrand()
+	case utils.ROLE_COMPANY_ADMIN:
+		api = config.GetHttpApiClient()
+	}
+
+	apiUrl := fmt.Sprintf(
+		"%s/my/active_calls/realtime_filter",
+		api,
+	)
```

Comment:
This uses an endpoint that does not exist in any API. Should we add here the commits required to make it work?

Reply:
It had been designed to add a future branch, which already exists with that endpoint.